### PR TITLE
ZEPPELIN-1867. Update document for pig interpreter and add one sample note

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?
* Minor update for pig interpreter
* Add one sample pig tutorial note which do the same thing as the spark tutorial note. 

### What type of PR is it?
[Improvement | Documentation ]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-1867

### How should this be tested?
Tested manually

### Screenshots (if appropriate)
image

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by zjffdu** [comment]
@felixcheung @AhyoungRyu Please help review.

File: docs/interpreter/pig.md [hunk 1]
```diff
@@ -94,4 +99,4 @@ c = group b by Category;
 foreach c generate group as category, COUNT($1) as count;
 ```
 
-Data is shared between `%pig` and `%pig.query`, so that you can do some common work in `%pig`, and do different kinds of query based on the data of `%pig`.
+Data is shared between `%pig` and `%pig.query`, so that you can do some common work in `%pig`, and do different kinds of query based on the data of `%pig`. There's one pig tutorial note in zeppelin for your reference.
```
Comment (felixcheung) [inline docs/interpreter/pig.md]: I think it's a bit odd to mention this in doc in the `pig.query` section? perhaps move this to under the parent `Example` section instead? how do we reference tutorial notebook in other documentation?

**Comment by AhyoungRyu** [comment]
@zjffdu Nice! 

In the past `Spark Tutorial` is the only tutorial note in Zeppelin included by default. But after this PR, 5 tutorial notes will be shown to Zeppelin new commers. Since some ppl can try this Pig tutorial before Spark one, more detailed description will be helpful to users than just saying like below 
screen shot 2017-01-02 at 5 15 33 pm


e.g. short description about Pig / you need to install `hadoop` before trying to run this tutorial note, and etc etc..  

There is good example ([Mahout tutorial note](https://github.com/apache/zeppelin/blob/master/notebook/2BYEZ5EVK/note.json#L4)) you can refer to.

**Comment by zjffdu** [comment]
@AhyoungRyu I have updated the tutorial note, please help review.

**Comment by zjffdu** [comment]
ping @AhyoungRyu , please check the updated screenshot

File: notebook/2C57UKYWR/note.json [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+      "apps": [],
+      "jobName": "paragraph_1483277250240_-480070728",
+      "id": "20161228-155036_1854903164",
+      "dateCreated": "Jan 1, 2017 9:27:30 PM",
+      "status": "READY",
+      "errorMessage": "",
+      "progressUpdateIntervalMs": 500
+    }
+  ],
+  "name": "Zeppelin Tutorial/Pig Tutorial",
```
Comment (AhyoungRyu) [inline notebook/2C57UKYWR/note.json]: How about "Zeppelin Tutorial/Basic Features(Pig)" like the other Spark note does ?

**Comment by AhyoungRyu** [comment]
Sorry for my late response. It looks nice! Except one minor suggestion, LGTM 👍

**Comment by zjffdu** [comment]
Thanks @AhyoungRyu , I rename the note, please check the updated screenshot in PR description.

**Comment by AhyoungRyu** [comment]
Thanks! Merge into master if there are no more comments.

**Pull request closed without merging**